tests/unit_tests/data_model/test_data_reader.py

```diff
@@ -97,11 +98,8 @@ def test_read_value_from_file_and_validate(caplog, tmp_test_directory):
         assert "Successful validation of yaml/json file" in caplog.text
 
     # schema explicitly given
-    schema_file = (
-        "https://gitlab.cta-observatory.org/cta-science/simulations/"
-        "simulation-model/model_parameters/-/raw/main/"
-        "schema/reference_point_altitude.schema.yml"
-    )
+    schema_dir = Path(__file__).parent / "../../../simtools/schemas/model_parameters/"
```

Comment:
I think the "../../../" is not very elegant and can easily be disrupted by any change on the path. I suggest we create a variable for schema_dir in the configurator, as we have for simtel_path. This would also solve the problem I pointed out above

Reply:
This should be relative paths - remember that we distribute simtools also as pip package and those files need to be findable. I don't think this is so bad (of course it breaks when we move the model_parameters; we will notice immediately).

There is another solution for package data suggested in https://setuptools.pypa.io/en/latest/userguide/datafiles.html . I still not find time to understand exactly the details (beyond this PR).